src/tools/extractor_common/CascHandles.cpp

```diff
@@ -190,6 +190,31 @@ CASC::Storage* CASC::Storage::Open(boost::filesystem::path const& path, uint32 l
     return storage;
 }
 
+CASC::Storage* CASC::Storage::OpenRemote(uint32 localeMask, char const* product)
+{
+    HANDLE handle = nullptr;
+    boost::filesystem::path cachDir = boost::filesystem::current_path() / "Cache";
```

Comment:
I recommend not naming it `Cache` but something like `CascCache` or `ExtractorCascCache` to avoid clashing with clients own `Cache` directory if ran from where the exes are